src/Evolution/Systems/Cce/Actions/ScriObserveInterpolated.hpp

```diff
@@ -211,12 +202,26 @@ struct ScriObserveInterpolated {
                       interpolation_manager->interpolate_and_pop_first_time();
                 },
                 make_not_null(&box));
-            ScriObserveInterpolated::transform_and_write<
-                tag, tag::type::type::spin, ParallelComponent>(
-                interpolation.second, interpolation.first,
-                make_not_null(&goldberg_modes), make_not_null(&data_to_write),
-                file_legend, l_max, observation_l_max, cache);
+            data_to_write[detail::ScriOutput<tag>::name()] =
+                transform_nodal_data(make_not_null(&goldberg_modes),
+                                     interpolation.second, interpolation.first,
+                                     l_max, observation_l_max, tag{});
           });
+
+      const std::string subfile_name{"Cce"};
```

Comment:
Is this what we want to name the subfile? This produces output in the H5 file like 
```
/Cce.cce                 Group
/Cce.cce/EthInertialRetardedTime Dataset {26451/Inf, 163}
/Cce.cce/News            Dataset {26451/Inf, 163}
/Cce.cce/Psi0            Dataset {26451/Inf, 163}
/Cce.cce/Psi1            Dataset {26451/Inf, 163}
/Cce.cce/Psi2            Dataset {26451/Inf, 163}
/Cce.cce/Psi3            Dataset {26451/Inf, 163}
/Cce.cce/Psi4            Dataset {26451/Inf, 163}
/Cce.cce/Strain          Dataset {26451/Inf, 163}
/src.tar.gz              Dataset {7757329}
```

which seems a bit redundant? Other options could be `/Bondi.cce`, `/Scri.cce`, `/FutureNullInfinity.cce`, `/SpECTRE.cce`.

Reply:
I agree. I think any are fine, except maybe `Bondi` to avoid confusion with the `BondiSachs` worldtube data we write out from GH that's input into CCE? Another option could be to append the worldtube radius? E.g. `SpectreR100.cce` in case we do want to run multiple CCE's simultaneously in the future.